app/master/build.py

```diff
@@ -523,6 +527,22 @@ def _delete_temporary_build_artifact_files(self):
         end_time = time.time() - start_time
         self._logger.info('Completed deleting artifact files for {}, took {:.1f} seconds.', self._build_id, end_time)
 
+    def _update_prometheus_metrics(self):
+        serialized_time_val, estimation_error_val = 0, 0
+        for subjob in self._all_subjobs_by_id.values():
+            for atom in subjob.atoms:
+                if atom.actual_time is None:
+                    self._logger.warning('Could not load atom.actual_time for atom {} build {}', atom.command_string, self._build_id)
+                    return
+                if atom.expected_time is None:
+                    self._logger.warning('Could not load atom.expected_time for atom {} build {}', atom.command_string, self._build_id)
+                    return
+                serialized_time_val += atom.actual_time
+                estimation_error_val += abs(atom.actual_time - atom.expected_time)
+
+        serialized_build_time_seconds.observe(serialized_time_val)
+        build_estimation_error.observe(estimation_error_val)
```

Comment:
I'm starting to wonder if build_estimation_error is very useful to have as a metric. I think it would definitely be useful on a per-build level (so it would be cool to add it to the API response), but I'm not sure if it's actionable on a global level. What do you think?